docs/UserGuide.adoc

```diff
@@ -33,12 +43,13 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`list`* : lists all medicines in inventory
-* *`add`* `n/Paracetamol c/3M Pharmaceuticals` : adds a new medicine named Paracetamol to the inventory
+* *`list`* : lists all medicines in the inventory
+* *`add`* `n/Paracetamol c/3M Pharmaceuticals` : adds a new medicine named `Paracetamol` to the inventory
 * *`delete`* `3` : deletes the 3rd entry shown in the current list
 * *`exit`* : exits the app
 
 .  Refer to <<Features>> for details of each command.
+.  Having trouble setting up MediTabs? You can refer to the <<Frequently Asked Questions (FAQ)>> for solutions to common problems you might encounter. Do you require further assistance? If so, you can also post your queries link:https://github.com/CS2103-AY1819S2-T12-3/main/issues[here] and our team would do our best to assist you.
```

Comment:
Interesting addition but if I am not wrong our FAQ does not address any such issues

Reply:
I was deciding between adding a separate section called Troubleshooting or put it under FAQ. I have not update that portion. I will do so in future commits and PR